general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
+00:1f.0 ISA bridge: Intel Corporation Device a2ca
+00:1f.2 Memory controller: Intel Corporation 200 Series/Z370 Chipset Family Power Management Controller
+00:1f.3 Audio device: Intel Corporation 200 Series PCH HD Audio
+00:1f.4 SMBus: Intel Corporation 200 Series/Z370 Chipset Family SMBus Controller
+00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection (2) I219-V
+01:00.0 VGA compatible controller: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+01:00.1 Audio device: NVIDIA Corporation TU116 High Definition Audio Controller (rev a1)
+01:00.2 USB controller: NVIDIA Corporation TU116 USB 3.1 Host Controller (rev a1)
+01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)
+```
+This is the ouput you are looking for
+`01:00.3 Serial bus controller [0c80]: NVIDIA Corporation TU116 [GeForce GTX 1650 SUPER] (rev a1)`
+Now check the firmware revision its on by typing ```ndvidia-smi```
+you should get an ouput like this
+Sun Aug 16 20:42:25 2020       
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 450.57       Driver Version: 450.57       CUDA Version: 11.0     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  GeForce GTX 165...  Off  | 00000000:01:00.0  On |                  N/A |
+|  0%   50C    P0    26W / 100W |    719MiB /  3895MiB |     17%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+
+(you specifically just need this to see if you have the latest available driver and if so then skip steps 9.-14.
+
+2. So to find the correct drivers select from the dropdown. Againb using my Geoforce 970 as an example
+        Product Type: Geoforce (whoever manufacters your GPU)
+        Product Series: Geoforce 900 Series (make sure you are not selecting the M unless your on a laptop)
+        Product Geoforce GTX 970 (Select what model of graphics card you have for your GPU)
+        Operating System: LINUX 64BIT (This is very important!!)        
+        Download Type: Select Linux Short Lived if your graphics card is a bit older.WHEN I SAY OLDER I MEAN 2 YEARS OR LESS.
+                       Select Linux Long Lived if your NVIDIA CARD IS FAIRLY NEW.
+        Click Search. This will automatically tell you your most updated driver you will need. Write or type this out somewhere so you can access it later.
+
+3. Now go to this link [NVIDIA Github patch](https://github.com/keylase/nvidia-patch) for patching the drivers.
```

Comment:
Make this optional your guide is currently more about patching, than just getting NVENC to run

Reply:
okay ill add a subsection in installing the patch.